src/app/app.component.scss

```diff
@@ -0,0 +1,58 @@
+
+
+
+:host {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  min-height: 100%;
+}
+
+:host ::ng-deep mat-sidenav-container.app-container {
+  display: flex;
+  flex: 1 1 auto;
+  height: 100%;
+  min-height: 0;
+  position: relative;
+}
+
+/* Ensure the sidenav itself stretches full height and behaves as a column */
```

Comment:
We don't usually write all these comments. 
in my opinion, some might be useful, but most of them explain rather basic CSS styling. 
Anyway, I think we should stay consistent with the rest of the code and don't introduce over explanatory comments. What do you think @lukestanley ?